docs/introduction/vocabulary.mdx

```diff
@@ -2,7 +2,7 @@
 title: "Vocabulary"
 sidebar_position: 3
 ---
-
+![scarf pixel](https://static.scarf.sh/a.png?x-pxid=01346572-0d57-4df3-8399-1425db913a0a)
```

Comment:
I don't like how this pixel adds a whole empty line to our pages, for instance:

![image](https://github.com/ConduitIO/conduit-site/assets/8320753/c23c9c47-a254-4fd5-9f73-7531c9230683)

Is there a way to make the pixel actually invisible? Otherwise I propose to move the pixel to the bottom of all pages, the empty line at the end won't be as noticable.